### PR TITLE
User story 'Create sql definitions'

### PR DESCRIPTION
**Pull request by andjey**
Implementing of DB schema has been done

### CODE REVIEW

File: ecomap/db/ecomap/schema/photo.sql [hunk 1]
```diff
@@ -0,0 +1,14 @@
+/*
+    Table for storage attached photos to Problem entities.
+    Has reference
+      :on user_id -> a user, who attached the photo
+      :on problem_id -> specific Problem entity which photos belongs to
+*/
+CREATE TABLE IF NOT EXISTS `photo` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) NOT NULL,     # each photo must have unique name according to it /PATH/name.jpg
```
Comment (lhalam) [inline ecomap/db/ecomap/schema/photo.sql]: only name without path

File: ecomap/db/ecomap/schema/comment.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS `comment` (
```
Comment (lhalam) [inline ecomap/db/ecomap/schema/comment.sql]: add description

File: ecomap/db/ecomap/schema/page.sql [hunk 3]
```diff
@@ -0,0 +1,11 @@
+CREATE TABLE IF NOT EXISTS `page` (
```
Comment (lhalam) [inline ecomap/db/ecomap/schema/page.sql]: add description

File: ecomap/db/ecomap/schema/problem.sql [hunk 4]
```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS `problem` (
```
Comment (lhalam) [inline ecomap/db/ecomap/schema/problem.sql]: add description

File: ecomap/db/ecomap/schema/resource.sql [hunk 5]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS `resource` (
```
Comment (lhalam) [inline ecomap/db/ecomap/schema/resource.sql]: add description

File: ecomap/db/ecomap/schema/role.sql [hunk 6]
```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS `role` (
```
Comment (lhalam) [inline ecomap/db/ecomap/schema/role.sql]: add description

File: ecomap/db/ecomap/schema/vote.sql [hunk 7]
```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS `vote` (
```
Comment (lhalam) [inline ecomap/db/ecomap/schema/vote.sql]: add description

File: .gitignore [hunk 8]
```diff
@@ -55,3 +55,5 @@ docs/_build/
 
 # PyBuilder
 target/
+
+*.sql~
```
Comment (lhalam) [inline .gitignore]: Add newline at end of file

File: ecomap/db/ecomap/data/user.sql [hunk 9]
```diff
@@ -0,0 +1,2 @@
+INSERT INTO `user` (first_name, last_name, email, password)
+VALUES ('admin', 'admin', 'admin.mail@gmail.com', '1111');
```
Comment (lhalam) [inline ecomap/db/ecomap/data/user.sql]: Add user "Анонім" with id(2)

**Comment by andjey** [comment]
@lhalam, i've added changes you mentioned before. Check it.

**Pull request closed without merging**